waspc/src/Wasp/AI/GenerateNewProject/Page.hs

```diff
@@ -26,20 +27,43 @@ import Wasp.AI.GenerateNewProject.Common
 import qualified Wasp.AI.GenerateNewProject.Common.Prompts as Prompts
 import Wasp.AI.GenerateNewProject.Entity (entityPlanToWaspDecl)
 import Wasp.AI.GenerateNewProject.Operation (Operation (opImpl, opPlan), OperationImpl (opJsImpl))
+import qualified Wasp.AI.GenerateNewProject.Operation as Operation
 import qualified Wasp.AI.GenerateNewProject.Plan as Plan
 import Wasp.AI.OpenAI.ChatGPT (ChatMessage (..), ChatRole (..))
 
+getAllPossibleImports :: [Operation] -> [Text]
+getAllPossibleImports operations = possibleUnchangingImports ++ map makeOperationImport operations
+  where
+    possibleUnchangingImports :: [Text]
+    possibleUnchangingImports =
+      [ "import logout from '@wasp/auth/logout';",
+        "import useAuth from '@wasp/auth/useAuth';",
+        "import { useQuery } from '@wasp/queries';",
+        "import { useAction } from '@wasp/actions';"
+      ]
+
+    makeOperationImport :: Operation -> Text
+    makeOperationImport operation =
+      [trimming|
+        import ${opName} from '@wasp/${opType}/${opName}';
+      |]
+      where
+        opName = T.pack $ Plan.opName $ opPlan operation
+        opType = case Operation.opType operation of
+          Operation.Action -> "actions"
+          Operation.Query -> "queries"
+
 generateAndWritePage ::
   NewProjectDetails -> FilePath -> [Plan.Entity] -> [Operation] -> [Operation] -> Plan.Page -> CodeAgent Page
-generateAndWritePage newProjectDetails waspFilePath entityPlans actions queries pPlan = do
-  page <- generatePage newProjectDetails entityPlans actions queries pPlan
+generateAndWritePage newProjectDetails waspFilePath entityPlans queries actions pPlan = do
```

Comment:
`actions` and `query` parameters were in a different order than arguments sent in `generateNewProject`, causing the prompt to look like this:
```
          Actions in our app:
           - { "name": "createBird",
            "jsImpl": "export const createBird = async (args, context) => {\n  if (!context.user) { throw new HttpError(401) }\n\n  const { breedId, name, pictureUrl } = args\n\n  const bird = await context.entities.Bird.create({\n    data: {\n      breed: { connect: { id: breedId } },\n      name,\n      pictureUrl,\n      user: { connect: { id: context.user.id } }\n    }\n  })\n\n  return bird\n}"
          }
           - { "name": "logout",
            "jsImpl": "export const logout = async (args, context) => {\n  context.logout()\n}"
          }

          Queries in our app:
           - { "name": "getBirds",
            "jsImpl": "export const getBirds = async (args, context) => {\n  if (!context.user) { throw new HttpError(401) }\n\n  return context.entities.Bird.findMany({\n    where: {\n      user: { id: context.user.id }\n    },\n    include: {\n      breed: true\n    }\n  })\n}"
          }
           - { "name": "getBreeds",
            "jsImpl": "export const getBreeds = async (args, context) => {\n  if (!context.user) { throw new HttpError(401) }\n\n  return context.entities.Breed.findMany()\n}"
          }
```
I've decided to settle on a convention: queries first, actions second.